templates/repo/issue/fields/textarea.tmpl

```diff
@@ -0,0 +1,6 @@
+<div class="field">
+	{{template "repo/issue/fields/header" .}}
+	{{/* FIXME: preview markdown result */}}
+	{{/* FIXME: required validation for markdown editor */}}
```

Comment:
`FIXME: preview markdown result` anything wrong with it, is it a blocker?

If there are more than one markdown editor, do they all work together (including the ^V image pasting)? Does the dropzone also work with them together?

Reply:
Actually, I didn't do anything about this, but they just work well.

During my local test, I can paste images to every markdown editor and the dropzone also work well.